src/app/(landing)/@about/page.tsx

```diff
@@ -1,5 +1,39 @@
-function About() {
-  return <div>about</div>;
+import React from 'react';
+import Container from '@/components/Container';
+import Flex from '@/components/Flex';
+import Text from '@/components/Text';
+import styles from './about.module.scss';
+
+function AboutPage() {
+  return (
+    <Container className={styles.frame}>
+      <Flex direction="column" className={styles.container} gap="2x-big">
+        <Text textType="heading-lg" textColor="secondary-900" textWeight="bold">
```

Comment:
Should have text align for the two texts

Reply:
I added Flex align component for both